packages/scandipwa/src/component/MyAccountDownloadable/MyAccountDownloadable.style.scss

```diff
@@ -0,0 +1,18 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+
+.MyDownloadableOrders {
+    @include mobile {
+        overflow-x: scroll;
+        -webkit-overflow-scrolling: touch;
+        width: calc(100vw - 24px);
+    }
+}
```

Comment:
This comes from a different PR: https://github.com/scandipwa/scandipwa/pull/3033
So if we merge #3033 first and #3034 will be there any conflicts?

Reply:
Sorry, wrong branch... undid commit